server/asset/eth/eth_test.go

```diff
@@ -646,7 +689,7 @@ func TestTxData(t *testing.T) {
 	addr := randomAddress()
 	data := encode.RandomBytes(5)
 	tx := tTx(gasPrice, value, addr, data)
-	goodCoinID := (&TxCoinID{TxID: tx.Hash()}).Encode()
```

Comment:
I don't know, maybe we still want to have a versioned coin ID. But from where I'm at this just seems like avoidable overhead. Communications for ETH appear to be simpler than utxo-based assets.